contracts/ERC721M.sol

```diff
@@ -365,6 +371,14 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         );
     }
 
+    /**
+     * @dev Sets the mint currency used for payment.
+     */
+    function setMintCurrency(address newMintCurrency) external onlyOwner {
```

Comment:
let's disable set mint currency, better just set from constructor

Reply:
Hmm I think it could make sense to also set from constructor, but what's the reason for removing the setter?

Reply:
it prevents admin accidentally changing the mint currency and can rug users